wa/output_processors/postgresql.py

```diff
@@ -0,0 +1,601 @@
+#    Copyright 2018 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import uuid
+import collections
+
+try:
+    import psycopg2
+    from psycopg2 import (connect,
+                          extras)
+    from psycopg2 import Error as Psycopg2Error
+except ImportError as e:
+    psycopg2 = None
+    import_error_msg = e.args[0] if e.args else str(e)
+
+from wa.utils import postgres_convert
+from wa import OutputProcessor, Parameter, OutputProcessorError
+from wa.utils.types import level
+from wa.framework.target.info import CpuInfo
+
+from devlib.target import KernelVersion, KernelConfig  # pylint: disable=wrong-import-order
+
+
+class PostgresqlResultProcessor(OutputProcessor):
+
+    name = 'postgres'
+    description = """
+    Stores results in an Postgresql database.
+
+    Please see the documentation for information about the schema.
+
+    This can also be easily ascertained from the postgres_schema.sql file.
+    """
+    parameters = [
+        Parameter('username', default='postgres',
+                  global_alias='postgres_username',
```

Comment:
Remove global aliases --  they're not appropriate here.

Reply:
Done